installation/multi_machine_aws.rst

```diff
@@ -41,7 +41,7 @@ Below, we explain in detail the steps required to setup a multi-node Citus clust
 
 .. raw:: html 
   
- <p>To begin, you can start a Citus cluster using CloudFormation by clicking <a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Citus&templateURL=https:%2F%2Fcitus-deployment.s3.amazonaws.com%2Faws%2Fcitus6%2Fcloudformation%2Fcitus-6.2.3.json" onclick="trackOutboundLink('https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Citus&templateURL=https:%2F%2Fcitus-deployment.s3.amazonaws.com%2Faws%2Fcitus6%2Fcloudformation%2Fcitus-6.2.3.json'); return false;">here</a>. This will take you directly to the AWS CloudFormation console.</p>
+ <p>To begin, you can start a Citus cluster using CloudFormation by clicking <a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Citus&templateURL=https:%2F%2Fcitus-deployment.s3.amazonaws.com%2Faws%2Fcitus6%2Fcloudformation%2Fcitus-7.0.0.json" onclick="trackOutboundLink('https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Citus&templateURL=https:%2F%2Fcitus-deployment.s3.amazonaws.com%2Faws%2Fcitus6%2Fcloudformation%2Fcitus-7.0.0.json'); return false;">here</a>. This will take you directly to the AWS CloudFormation console.</p>
```

Comment:
The directory name here needs to be changed from citus6/ to citus7/. The link as-is gives me an error saying the template isn't found.